fiatconnect-api.md

```diff
@@ -141,7 +141,7 @@
           - [3.4.3.2.2.1. HTTP `200`](#343221-http-200)
         - [3.4.3.2.3. Semantics](#34323-semantics)
           - [3.4.3.2.3.1. Success](#343231-success)
-      - [3.4.3.3. `DELETE /accounts/:fiatAccountId`](#3433-delete-accountsfiataccountid)
+      - [3.4.3.3. `DELETE /accounts/delete/:fiatAccountId`](#3433-delete-accountsfiataccountid)
```

Comment:
I'm not sure about what's the benefit of having `delete` in the path?

Reply:
Swagger cannot differentiate between `/accounts/{fiatAccountId}` and `/accounts/{fiatAccountSchema}` as they are equivalent paths.